Tasks/AzureResourceGroupDeployment/task.json

```diff
@@ -379,8 +379,10 @@
         "AddingExtensionFailed": "Addition of extension on vm %s failed",
         "TimeoutWhileWaiting": "Timed out while waiting",
         "InvalidTemplateLocation": "The template location supplied is invalid. Task only supports 'Linked artifact' or 'URL of the file'",
-        "EncodingNotSupported": "Encoding of the file '%s' is '%s' which is not supported. Supported encodings are ['utf-8', 'utf-16le']",
+        "EncodingNotSupported": "Encoding of the file '%s' is '%s' which is not supported",
         "CouldNotDetectEncoding": "Could not detect encoding of file '%s'",
+        "DetectingOnlyUtf": "Supported file encodings are ['utf-8', 'utf-16le']. The deployment is subject to potential failures for other formats", 
+        "DetectedFileEncoding": "The detected file encoding for '%s' is '%s'",
```

Comment:
lets have better prefix so that loc team has easier job while translating. The detected encoding for file '%s' is '%s'